var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -90,9 +90,26 @@ class Gromacs(CMakePackage, CudaPackage):
         default=False,
         description="Produces a double precision version of the executables",
     )
-    variant("cufftmp", default=False, when="+cuda+mpi", description="Enable Multi GPU FFT support")
+    variant(
+        "cufftmp",
+        default=False,
+        when="@2022: +cuda+mpi",
+        description="Enable multi-GPU FFT support with cuFFTMp",
+    )
+    variant(
+        "heffte",
+        default=False,
+        when="@2021: +sycl+mpi",
```

Comment:
Technically GROMACS supports HeFFTe also with `+cuda+mpi` but I don't know how to express a variant that can be satisfied by mutually exclusive dependencies, and nobody would use such a GROMACS build, anyway.

Reply:
@mabraham I suppose it could be useful in cases when there are issues with cufftmp (e.g. like https://manual.gromacs.org/2023.3/install-guide/index.html#using-cufftmp). Still - I think this could be explored and added later in order to not hold up adding these variants (for the more common cases).

Reply:
Let's add that later then!